### PR TITLE
Format target added and Improved test target

### PR DESCRIPTION
**Pull request by harens**
[black](https://github.com/psf/black) has been used to format the python files, whilst [go fmt](https://golang.org/cmd/go/#hdr-Gofmt__reformat__package_sources) has been used for the go files.

Tests have also been written to make sure that the files have been formatted accordingly following a PR/commit.

### CODE REVIEW

**Comment by harens** [comment]
Forgot that version/version.go is auto-generated, and so not formatted

File: tools/make_release_tags.py [hunk 1]
```diff
@@ -239,10 +284,14 @@ def handle_repo_noexc(repo_name, from_tag, to_tag):
     if len(new_tasks) == 0:
         break
     else:
-        print("%d active (%s%s)" % (
-            len(new_tasks),
-            " ".join(r[0] for r in new_tasks[:5]),
-            " ..." if len(new_tasks) > 5 else ""))
+        print(
```
Comment (rojer) [inline tools/make_release_tags.py:287]: i don't like this.
```
it's
    so
            damn
        verbose
```

File: cli/bindata.go [hunk 2]
```diff
@@ -14,14 +14,15 @@
 package main
 
 import (
-	"github.com/elazarl/go-bindata-assetfs"
 	"fmt"
+	"github.com/elazarl/go-bindata-assetfs"
```
Comment (rojer) [inline cli/bindata.go:18]: please split system imports from external, this kind of grouping doesn't make sense.

**Review by rojer — CHANGES_REQUESTED** [review]

**Comment by harens** [comment]
Thanks @rojer for the review. With second thoughts I think that this is too large a PR anyway, and it might be better to have smaller PRs for each change.

**Pull request closed without merging**